eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/recommend/EventMeshRecommendImpl.java

```diff
@@ -31,179 +31,233 @@
 import java.util.List;
 import java.util.Map;
 
-import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-@Slf4j
```

Comment:
hi, @jonyangx , `slf4j` log annotation here was introduced in this PR #2761 recently by @weihubeats , may I know why do you want to remove it? 
maybe we need to discuss and make a consensus to use a unified solution for the whole project.

Reply:
lombok is not good for ide. @horoc

Reply:
I think since we use lombok we can use @ slf4j to simplify the log printing

Reply:
Actually, we do have lots of Lombok annotations in the project, like `@Data`, `@Builder`, `@Slf4j`,  and as @jonyangx mentioned, it's true that there is some shortage of Lombok annotation, the developers need to do extra configuration in IDE to enable Lombok parsing. But in the meantime, Lombok also has provided its own convenience for developers.

I think the key point is, we need to keep a consistent code style for the whole project, otherwise,  we may modify the same place back and forth by different developers. 😂

Reply:
OK, I could fix it using lombok,but it's not good for vim/ultraedit ide.

Reply:
@weihubeats  how to simplify the log printing?

Reply:
I don't think so.

Reply:
> OK, I could fix it using lombok,but it's not good for vim/ultraedit ide.

@jonyangx , I think your concern about the inconvenience of developers who use other IDE is reasonable.

In my point of view, if you think it's better to remove Lombok dependencies,  you can raise a new issue to discuss this point, and after discussion, if it's really necessary, we can do an optimization in a new pr to remove all Lombok annotations.

Reply:
If we want to print the log we just need to add the comment @Slf4j, without writing similar code
`
private static final Logger LOG = LoggerFactory.getLogger(WeightRoundRobinLoadBalanceSelector.class);
`
In practice, sometimes we will find some complex code xx.class in the project is wrong, and not the actual printing log class, so that we can reduce the workload of printing logs and improve development efficiency

Reply:
If it is not friendly to the development tools vim/ultraedit, how should the heavy use of lombok in the project be handled? Should the entire code style be consistent, either by removing lombok dependencies or by using lombok uniformly?